x/ethbridge/client/cli/tx.go

```diff
@@ -271,17 +271,22 @@ func GetCmdUpdateWhiteListValidator() *cobra.Command {
 				return err
 			}
 
-			validatorAddress, err := sdk.ValAddressFromBech32(args[1])
+			networkID, err := strconv.Atoi(args[1])
+			if err != nil {
+				return errors.New("Error parsing network descriptor")
+			}
+
+			validatorAddress, err := sdk.ValAddressFromBech32(args[2])
 			if err != nil {
 				return err
 			}
 
-			operationType := args[2]
```

Comment:
Why did we remove operationType?

Reply:
now we have the power for each validator, just set the power as zero to disable the validator.